src/screens/AssignmentList/NewAssignmentScreen.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import {
+  View, Button, Text, ActivityIndicator, StyleSheet,
+} from 'react-native';
+import { TextInput } from 'react-native-gesture-handler';
+import PropTypes from 'prop-types';
+import Firestore from '@react-native-firebase/firestore';
+import Firebase from '@react-native-firebase/app';
+import { Calendar } from 'react-native-calendars';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    padding: 20,
+
+  },
+  postContainer: {
+    flex: 1,
+
+  },
+});
+
+export default function NewAssignmentScreen({ navigation }) {
```

Comment:
This code is essentially code from NewPostScreen.js right? I think rather than copying the code and making minor changes, we should modify the Post component to retrieve documents from either the Assignments or Announcement collection respectively (perhaps pass parameters to the NewPostScreen that indicate whether this is an Assignment or Announcement Post). You can make modifications to the NewPostScreen instead of creating a new essentially duplicate file.